site/layouts/partials/docs-navbar.html

```diff
@@ -1,39 +1,40 @@
-      <img src="/docs/{{ .Site.Params.docs_version }}/assets/img/orange-logo.svg" width="50" height="50" alt="Back to homepage" title="Go to homepage" />
-    </a>
-
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#bdNavbar" aria-controls="bdNavbar" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse" id="bdNavbar">
-      <ul class="navbar-nav flex-row flex-wrap pt-2 py-md-0">
-        <li class="nav-item col-6 col-md-auto">
-          <a class="nav-link{{ if .IsHome }} active" aria-current="page{{ end }}" href="/" onclick="dataLayer.push({'event': 'clic', 'site_name':'accessibility-boosted', 'phase':'prod', 'track_category':'navbar', 'track_name':'home'});">Home</a>
-        </li>
-        <li class="nav-item col-6 col-md-auto">
-          <a class="nav-link{{ if eq .Page.Layout "docs" }} active" aria-current="true{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/getting-started/introduction/" onclick="dataLayer.push({'event': 'clic', 'site_name':'accessibility-boosted', 'phase':'prod', 'track_category':'navbar', 'track_name':'docs'});">Docs</a>
-        </li>
-        <li class="nav-item col-6 col-md-auto">
-          <a class="nav-link{{ if eq .Page.Title "Examples" }} active" aria-current="true{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/examples/" onclick="dataLayer.push({'event': 'clic', 'site_name':'accessibility-boosted', 'phase':'prod', 'track_category':'navbar', 'track_name':'examples'});">Examples</a>
-        </li>
-        <li class="nav-item col-6 col-md-auto">
-          <a class="nav-link{{ if eq .Page.Title "Orange Design System for web" }} active" aria-current="true{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/guidelines/" onclick="dataLayer.push({'event': 'clic', 'site_name':'accessibility-boosted', 'phase':'prod', 'track_category':'navbar', 'track_name':'guidelines'});">Design guidelines</a>
-        </li>
-      </ul>
-
-      <ul class="navbar-nav flex-row flex-wrap ms-md-auto">
-        <li class="nav-item col-6 col-md-auto py-3 py-md-0">
-          <a class="nav-link" href="{{ .Site.Params.repo }}" target="_blank" rel="noopener" aria-label="GitHub">
-            {{ partial "icons/github.svg" (dict "class" "navbar-nav-svg d-inline-block align-text-top" "width" "20" "height" "20") }}
-          </a>
-        </li>
-        <li class="nav-item col-6 col-md-auto py-3 py-md-0 align-self-center">
-          <a class="btn btn-primary btn-inverse my-2 ms-2 my-md-0 ms-md-3" href="/docs/{{ .Site.Params.docs_version }}/getting-started/download/">Download</a>
-        </li>
-      </ul>
+<header class="sticky-top">
+  <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Main navigation">
+    <div class="container-xxl">
+      <a class="navbar-brand order-first me-auto me-lg-4" href="/" aria-label="Boosted">
```

Comment:
`.order-first` is not in the Global Headers examples. Useful here?

Reply:
Thanks for the modification. Here are some discrepancies between the GHs and what's contained here.
* Here we have `<a class="stretched-link" href="/" aria-label="Boosted">` but in http://localhost:9001/docs/5.1/examples/navbar-sticky/, the same link doesn't have an `aria-label`.
* Here `<img src="/docs/5.1/assets/brand/orange-logo.svg" width="50" height="50" role="img" alt="Back to homepage" title="Go to homepage" loading="lazy">`:
  * has a `title` but in http://localhost:9001/docs/5.1/examples/navbar-sticky/, the same image doesn't have a `title`.
  * has a `role="img"` but in http://localhost:9001/docs/5.1/examples/navbar-sticky/, there's not such role (is it useful?)
* Here we have `<a class="nav-link nav-icon" href="{{ .Site.Params.repo }}" target="_blank" rel="noopener" aria-label="GitHub">` but in http://localhost:9001/docs/5.1/examples/navbar-sticky/, link icons don't have an `aria-label`
* In the search link in the GHs (http://localhost:9001/docs/5.1/examples/navbar-sticky/) we have a `<span class="visually-hidden">Search</span>` but not here.

Reply:
For the last point, there's no need for visually-hidden span since the svg got a title so it's vocalized the good way as-is